app/assets/locales/messages_ccodk_default.txt

```diff
@@ -451,6 +451,7 @@ problem.report.menuitem=Report Problem
 verification.title=Verifying Multimedia
 verification.checking=Verifying...
 verification.progress=Checked ${0} out of ${1} resources
+verification.success.message=Application's multimedia completely installed

```

Comment:
The word 'completely' feels strange; can this say 'Your application's multimedia is installed' or '... is valid'?